content/en/_index.html

```diff
@@ -163,8 +163,9 @@
     <div class="container">
       <div class="row">
         <div class="col-xs-12 col-lg-12">
-          <div class="tag-line">Accelerate Your Continuous Delivery on Kubernetes</div>
-          <div class="tag-line-secondary mx-auto">automate the continuous delivery of change through your environments via GitOps and create previews on Pull Requests to help you accelerate</div>
+          <div class="tag-line">All In One CI/CD for Kubernetes</div>
```

Comment:
May be `Batteries included` or something with batteries included? I honestly think we need to emphasize we are a CI/CD platform and not a tool, as we do a lot more things than a tool (cert manager, ex dns, ingress even service mesh etc ...)

Reply:
I modified the text a bit. It is hard because it is much snappier and pretty when the text is short....